pages/episodes-page.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * NOTE: This file contains code related only to the "episodes" page
+ */
+
+// missing image placeholder
+const no_img_link_e =
+  "http://smithssanitationsupply.ca/wp-content/uploads/2018/06/noimage-1.png";
+
+// this function is called when a user selects a show from shows dropdown list (shows or episodes page)
+// or when user clicks on the title of a show from the shows display (shows page)
+function getAllEpisodes(showId, showsList) {
+  const episodesLink = `https://api.tvmaze.com/shows/${showId}/episodes`;
+  fetch(episodesLink)
+    .then((response) => response.json())
+    // on successful response
+    .then((data) => makePageForEpisodes(data, showsList, showId))
+    // on failed response
+    .catch(() => createInitialContent("error"));
```

Comment:
I would use cosole.error as this would capture the error a display it in the console

Reply:
Thanks @SenaitEkubai for your feedback.
I put that code in line 18 to turn the page into one that is similar to "Error: 404" in an event the episodes page fails to load. 
The thing is, if I used console.error(), the user wouldn't know an error occurred, unless they open "dev tools". That's what I thought anyway.

Reply:
Hi @cbaggini, 
Thank you for your feedback. It's comprehensive and well organised, I really like it! :-) Thank you for taking the time to review my work, too.
That being said, I have already started making improvements that hopefully address the issues mentioned (and maybe more). Cheers. :-)